src/core/workflow.py

```diff
@@ -23,6 +23,20 @@
     'prepublication': [submission_models.STAGE_READY_FOR_PUBLICATION]
 }
 
+STAGES_ELEMENTS = {
+    submission_models.STAGE_ASSIGNED: 'review',
+    submission_models.STAGE_UNDER_REVIEW: 'review',
+    submission_models.STAGE_UNDER_REVISION: 'review',
+
+    submission_models.STAGE_EDITOR_COPYEDITING: 'copyediting',
+    submission_models.STAGE_AUTHOR_COPYEDITING: 'copyediting',
+    submission_models.STAGE_FINAL_COPYEDITING: 'copyediting',
+
+    submission_models.STAGE_TYPESETTING: 'typesetting',
+    submission_models.STAGE_PROOFING: 'proofing',
+    submission_models.STAGE_READY_FOR_PUBLICATION: 'prepublication',
+}
```

Comment:
We could generate this from ELEMENTS_STAGES to be a bit more DRY but I'll leave that as optional.